_docs/operations/managing-host-storage.md

```diff
@@ -1,43 +1,51 @@
-recommended to use specific disk devices only for StorageOS.
+StorageOS uses the storage available on the nodes where it is installed to
+present as available for volumes.
 
-StorageOS needs a filesystem formatted and mounted to `/var/lib/storageos/data`. It is possible to
-use different disks or partitions to expand StorageOS' available space. By default, the directory
-`/var/lib/storageos/data/dev1` will be created when volumes are used. However, it is possible to
-shard the data files by creating more directories alike. StorageOS will save data in any directory
-that follows the name `dev[0-9]+`, such as `/var/lib/storageos/data/dev2` or
-`/var/lib/storageos/data/dev5`. This functionality enables operators to mount different devices into
-devX directories and StorageOS will recognise them as available storage automatically.
+In order to mitigate against problems caused by filling the host root disk, we
+recommend mounting a separate device into the `/var/lib/storageos` directory.
+StorageOS is agnostic to the type of filesystem mounted in
+`/var/lib/storageos`.
 
-These are 2 possible options to expand the available disk space for StorageOS to allocate (combinations of
-these options are also possible):
+## Extending Available Storage
 
-1. Mount filesystem in `/var/lib/storageos/data/devX` (available on next release)
+StorageOS uses subdirectories of `/var/lib/storageos/data` to hold user data.
+By default, the directory `/var/lib/storageos/data/dev1` will be created when a
+node is bootstrapped, and used for pool data. It is possible to shard the data
+by creating more directories into this structure. StorageOS will save data in
+any directory that conforms to the pattern `/var/lib/storageos/data/dev[0-9]+`,
+such as `/var/lib/storageos/data/dev2` or `/var/lib/storageos/data/dev5`. This
+functionality enables operators to mount different devices into devX
+directories and StorageOS will recognise them as available storage
+automatically.
+
+There are two possible options to expand the available disk space for StorageOS
+to allocate:
+
+1. Mount filesystem in `/var/lib/storageos/data/devX`
 1. Use LVM to expand the logical volume available to StorageOS
 
-## Option 1: Mount filesystem (available on next release)
+## Option 1: Mount Additional Devices
 
-This option enables operators to expand the cluster's available space at any time without having to stop any service or forcing operational downtime.
-The expansion of disk is transparent for applications or StorageOS Volumes. StorageOS will use the new
-available space to create new data files in case that the node run out of disk space. By expanding
-the number of disks that serve data, the throughput also increases as the data of one StorageOS
-volume can be segmented in different physical devices.
+This option enables operators to expand the cluster's available space at any
+time without having to stop any service or forcing operational downtime. The
```

Comment:
```suggestion
time without having to stop any services or forcing operational downtime. The
```

Reply:
Hmm, funny, I'd go with singular for that one. 'any service' seems right?

Reply:
Reworded slightly.